### PR TITLE
Feature/nats in k8s

### PR DESCRIPTION
**Pull request by ajax-makarytskyi-o**
Nats was added to k8s and docker.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
           fetch-depth: 0
 
       - name: Start NATS container
-        run: docker compose -f docker-compose-github-actions.yml up -d
+        run: docker compose -f docker-compose.yml --profile test up -d
```
Comment (ajax-marchenko-s) [inline .github/workflows/build.yml]: you could remove `-f docker-compose.yml`

**Review by ajax-marchenko-s — COMMENTED** [review]
Great work!

File: k8s/nats.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nats-depl
+  labels:
+    app: nats
+spec:
```
Comment (ajax-shulepa-a) [inline k8s/nats.yaml:7]: how many replicas of nats do you have?

**Review by ajax-shulepa-a — APPROVED** [review]

File: k8s/nats.yaml (hunk 2 above)
Comment (ajax-makarytskyi-o) [reply k8s/nats.yaml:7]: Here is only 1 nats replica.

**Pull request merged**